docs/website/docs/overview/installation.md

```diff
@@ -248,28 +252,57 @@ echo 'export PATH=$PATH:$HOME/bin' >> ~/.bashrc
 
 ---
 
-## Windows
+### Windows
 
 1. Open a PowerShell terminal
 
 2. Download the latest release from the mirror:
 ```shell
-curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.0.0/odo-windows-amd64.exe -o odo.exe
+curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.3.0/odo-windows-amd64.exe -o odo.exe
 ```
 
 2. (Optional) Verify the downloaded binary with the SHA-256 sum:
 ```shell
-curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.0.0/odo-windows-amd64.exe.sha256 -o odo.exe.sha256
+curl -L https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.3.0/odo-windows-amd64.exe.sha256 -o odo.exe.sha256
 # Visually compare the output of both files
 Get-FileHash odo.exe
 type odo.exe.sha256
 ```
 
 4. Add the binary to your `PATH`
 
+## IDE Installation
+
+### Visual Studio Code (VSCode)
+
+The [OpenShift Toolkit](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector) VSCode extension uses both `odo` and `oc` binaries
+to interact with Kubernetes or OpenShift cluster.
+
+To install it:
+
+1. Open VS Code.
+2. Launch VS Code **Quick Open** (`Ctrl+P`).
+3. Paste the following command:
+
+```
+ext install redhat.vscode-openshift-connector
+```
+
+More details in the [following demo](https://www.youtube.com/watch?v=HEsYgDqD1rM).
```

Comment:
This demo is an old one. But since there's no new demo, we can ask the adapters team if they can create a new one. We don't wnat to wait on that, so you can probably add a note saying that it's an old one.

Reply:
I saw it as a nice overview of the plugin capabilities (even if it is old, the base idea of how to use it was there).
But ok, I've removed it for now. We can consider adding a link once there is a newer demo available.

Reply:
>I saw it as a nice overview of the plugin capabilities (even if it is old, the base idea of how to use it was there).

I agree with that. I was mostly suggesting we keep it but add a note that it is an old demo.

Reply:
With hindsight, I think it might make it more confusing to state that it is an old demo.

We are anyway referencing the [extension page on the Marketplace](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector). And this page already contains a demo (outdated, but I think they have an issue to update it). So up to the Adapters team to update that page instead.

Reply:
Okay. I have requested the adapters team to share the new demo with us as soon as they have it. It's already in progress.  We can update our docs once it's ready.